src/components/ebay-carousel/index.js

```diff
@@ -118,6 +118,7 @@ function getTemplateData(state) {
         offset: hasOverride ? config.offsetOverride : offset,
         disableTransition: hasOverride,
         showPaddles: itemsPerSlide ? items.length > itemsPerSlide : true,
+        alwaysShowPaddles: state.noDots && state.peek === 0 && !state.autoplayInterval,
```

Comment:
@agliga Why do we need a check for `autoplayInterval`?

Reply:
This was brought up by Ian in a previous comment. 
The idea behind this fix is if we have a carousel with no peek and no dots we should show that this widget is a carousel. Otherwise people won't know unless they hover over it.
So when we have an autoplay carousel, we shouldn't show paddles because the autoplay will show that it is a carousel and automatically switch to the next slide and show the play/pause button.